### DISCUSSION TITLE
tiptap react, using textarea element as my editor

### DISCUSSION THREAD

**Discussion by uibahy** [ueberdosis/tiptap#3964] [General Questions]
how can i use <textarea></textarea> as my editor?
```
<div className="tw-flex tw-flex-col">
      <MenuBar editor={editor} />
      <textarea
        className="form-control element"
        id="my-input"
      ></textarea>
      // my current editor
      <EditorContent editor={editor} />
</div>

```

**Comment by azkaban90** [comment]
has resolved?

**Reply by uibahy** [reply]
not yet, I just used Tiptap default editor

**Comment by nonameolsson** [comment]
I am also interested in this. To solve it I am thinking of just adding my css styles to mimic the other input fields. But I would love to be able to use my existing textarea-component for this.

**Comment by CoHarsh** [comment]
Any update on this?
